ddp-workspace/projects/ddp-basil/src/assets/i18n/es.json

```diff
@@ -23,287 +23,287 @@
+            "PasswordRequiredError": "[ES] Password is required"
         },
         "ErrorPage": {
-            "Title": "We've encountered an error",
-            "Text": "We're sorry, but an error has occurred.",
+            "Title": "[ES] We've encountered an error",
+            "Text": "[ES] We're sorry, but an error has occurred.",
             "DefaultText": {
-                "TextPt1": "Please wait a moment and retry. If the trouble persists, please email us at",
-                "TextPt2": "or call us at"
+                "TextPt1": "[ES] Please wait a moment and retry. If the trouble persists, please email us at",
+                "TextPt2": "[ES] or call us at"
             }
         },
         "SessionExpired": {
-            "Text": "You were away for a while, so we logged you out as a security precaution. Please log in again to return to the last page you were on."
+            "Text": "[ES] You were away for a while, so we logged you out as a security precaution. Please log in again to return to the last page you were on."
         },
         "Dialogs": {
             "SessionWillExpire": {
-                "Title": "Your session will time-out soon",
-                "Text": "To protect your privacy, the system is going to log you out because you have been away for a while. Click “Continue” to stay logged in. Click “Sign Out” if you are done for now.",
-                "SignOut": "Sign Out",
-                "Continue": "Continue",
-                "RenewalFailed": "Session renewal failed. Please log out and log back in again."
+                "Title": "[ES] Your session will time-out soon",
+                "Text": "[ES] To protect your privacy, the system is going to log you out because you have been away for a while. Click “Continue” to stay logged in. Click “Sign Out” if you are done for now.",
+                "SignOut": "[ES] Sign Out",
+                "Continue": "[ES] Continue",
+                "RenewalFailed": "[ES] Session renewal failed. Please log out and log back in again."
             }
         }
     },
     "SDK": {
-        "SubmitButton": "Submit",
-        "CloseButton": "Close",
-        "AddButton": "ADD",
-        "CancelButton": "Cancel",
-        "NextButton": "Next",
-        "PreviousButton": "Back",
-        "SavingButton": "Saving",
-        "EditButton": "Continue",
-        "SaveButton": "Save",
-        "CompleteButton": "Edit",
-        "ReviewButton": "View",
-        "OpenButton": "Open",
-        "StartButton": "Start",
-        "AgreeButton": "I agree",
-        "ReportButton": "Report",
-        "NotAgreeButton": "I'm not ready to agree",
-        "ValidateError": "Please complete all required fields.",
-        "CommunicationError": "There was a problem saving your data. Please check the form and your answers before submitting",
-        "SaveError": "We're sorry but an error occurred",
+        "SubmitButton": "[ES] Submit",
```

Comment:
I see here we prefix it with [ES] but for the RUS part it uses the actual tranlastion, does it work both ways?

Reply:
Don't think we want to invest tons of time on this right now, but we do have a lot of these translations in other projects.
It would. be worthwhile spending 30mins copying and pasting some of them. E.g.: We definitely have the months of the year in Spanish somewhere@

Reply:
Basil is a sandbox project. It is used for debug new features usually.
Correct translations are not needed. 
I added [ES]/ [RU] prefixes for all the strings just to be able to differ them on UI. 
Then I translated some strings in Russian just for fun. It was not a big deal for me. 

For production projects we ask we request a professional translation from a special team/service.

Reply:
Anyway, 8ef06d92f98f474f99d360ba4aede2958488bdf4

Reply:
Nice to know, thank you